imsv-docs-docusaurus/openapi/endpoints/prerequisites/spending-prerequisites.yaml

```diff
@@ -147,6 +147,9 @@ post:
 
         **UNSUPPORTED_KYC_REGION**
         The KYC region is not yet supported.
+
+        **KYC_STATEMENT_REGION_MISMATCH**
+        The latest KYC statement is either not submitted or requires manual review, and the region is different from the requested region.
```

Comment:
there are 2 issues with this wording:
- there is a logical error: not submitted or requires manual review -> not submitted or **does not** require manual review
- even with the above correction it is looking to complex to understand, suggesting a simpler wording:
"The latest unchecked KYC statement is for a region that is different from the requested region."